google/cloud/bigquery/v2/minimal/internal/common_v2_resources.h

```diff
@@ -316,6 +316,7 @@ bool operator==(Struct const& lhs, Struct const& rhs);
 
 struct ColumnData {
   std::string value;
+  bool is_null{false};
```

Comment:
I think it would be more idiomatic to use `absl::optional<std::string>` to represent this.  At the very least it would avoid awkward situations like `value == "foo"` and `is_null == true`.